testinfra/plugin.py

```diff
@@ -250,7 +250,9 @@ def write(self, s):
 @pytest.mark.trylast
 def pytest_configure(config):
     if config.option.verbose > 1:
-        logging.basicConfig()
```

Comment:
Pytest supports logging, why even configure it in the first place?

Reply:
Hmm. Makes more sense. Perhaps only do it for those versions then.

Reply:
I think it's ok to not support this on older pytest versions.